varsim.py

```diff
@@ -127,7 +126,7 @@ def monitor_multiprocesses(processes, logger):
     for p in processes:
         p.join()
         if p.exitcode != 0:
-            logger.error("Process with pid %d failed with exit code %d" % (p.pid, pid.exitcode))
+            logger.error("Process with pid %d failed with exit code %d" % (p.pid, pid.exitcode)) # Marghoob: pid?
```

Comment:
@marghoob pid does not exist? I don't quite understand this. please help :8ball: